inference-engine/tests/functional/plugin/gpu/remote_blob_tests/cldnn_remote_blob_tests.cpp

```diff
@@ -84,6 +84,61 @@ TEST_F(RemoteBlob_Test, smoke_canInputUserBlob) {
     }
 }
 
+
+TEST_F(RemoteBlob_Test, smoke_canInputPluginRemoteBlob) {
+#if defined(ANDROID)
+    GTEST_SKIP();
+#endif
+    CNNNetwork net(fn_ptr);
+
+    net.getInputsInfo().begin()->second->setLayout(Layout::NCHW);
+    net.getInputsInfo().begin()->second->setPrecision(Precision::U8);
+
+    // TODO: Issue: investigate issue with IECore
```

Comment:
do we have a ticket for this?

Reply:
@iefode As I can see, this TODO was added by your patch. Do you remember the details?

Reply:
The issue was the using PluginCache